Tests/run_tests.py

```diff
@@ -55,9 +55,11 @@
 # The default verbosity (not verbose)
 VERBOSITY = 0
 
-# Following modules have historic failures. If you fix one of these
-# please remove here!
-EXCLUDE_DOCTEST_MODULES = []
+# Bio.Alphabet has been removed from Biopython. Importing it will raise an
+# informative RuntimeException.
+EXCLUDE_DOCTEST_MODULES = [
+    "Bio.Alphabet"
+]
```

Comment:
Comment needs to change to ImportError - or can we remove this change now?

Reply:
I prefer to keep it in EXCLUDE_DOCTEST_MODULES, as this doctest is never supposed to run.